index.js

```diff
@@ -1,18 +1,110 @@
-const readline = require('readline');
+const readline = require("readline");
 const rl = readline.createInterface(process.stdin, process.stdout);
+// Valid string inputs -----------------------------
+const correctAnswer = "yes" || "y" || "ys";
+const wrongAnswer = "no" || "n";
+const highAnswer = "higher" || "h" || "high";
+const playComputerGuessing = 'computer';
+const playPlayerGuessing = 'player'
 
+// Function list -----------------------------------
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
+} // Sanatizes string input
+function sanitizeString(string) {
+  string = string
+    .toString()
+    .trim()
+    .toLowerCase();
+  return string;
+} // Creates a random integer between two numbers
+function randomInteger(min, max) {
+  return Math.floor(Math.random() * (max - min)) + min;
```

Comment:
Here your randomInteger function is a little bit off and will give an off by one error. If you modify it to look like the function [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#Getting_a_random_integer_between_two_values_inclusive) then you should be good to go